src/monitor/Qml/AgentsView.qml

```diff
@@ -165,9 +172,19 @@ Item {
                 }
 
                 delegate: Rectangle {
-                    implicitWidth: 150
-                    implicitHeight: 20
-                    border.width: 1
+                    id: delegateRectangle
+                    required property var row
+                    implicitWidth: 100
+                    border.width: 0
+                    color: {
```

Comment:
How about ternary?

Reply:
Well, if this make any difference here why not.